guarddog/cli.py

```diff
@@ -165,14 +165,18 @@ def _scan(identifier, version, rules, exclude_rules, output_format, exit_non_zer
     if scanner is None:
         sys.stderr.write(f"Command scan is not supported for ecosystem {ecosystem}")
         exit(1)
-    results = {}
+    results = []
     if is_local_target(identifier):
         log.debug(f"Considering that '{identifier}' is a local target, scanning filesystem")
-        results = scanner.scan_local(identifier, rule_param)
+        if os.path.isdir(identifier):
+            for package in os.listdir(identifier):
```

Comment:
what's the expected format of each of the directory entries? files? directory? works for both? we should make it explicit

Reply:
It works for both. If it's found a local_target with the previous if statement `if is_local_target(identifier):`, we test if it's a directory.
What kind of explicit details would you like to see?